src/Lister/SimpleSubPageFinder.php

```diff
@@ -72,7 +72,24 @@ public function setLimit( $limit ) {
 
 		$this->setOption( self::OPT_LIMIT, $limit );
 	}
+	
+	/**
+	 * @see SubPageFinder::setRedirects
+	 *
+	 * @since 1.3.1
+	 *
+	 * @param bool $redirects
+	 *
+	 * @throws InvalidArgumentException
+	 */
+	public function setRedirects( $redirects ) {
```

Comment:
This name suggests the method sets a list of redirects. Same goes for the parameter name. How about "include redirects"?